docs/alerts/monitors/monitor-faq.md

```diff
@@ -6,6 +6,16 @@ description: Frequently asked questions about Sumo Logic monitors.
 
 import AlertsTimeslice from '../../reuse/alerts-timeslice.md';
 
+## How can I optimize scan costs for monitors when using Flex Pricing?
+
+To optimize scan costs for monitors under [Flex Pricing](/docs/manage/partitions/flex), consider the following factors:
+
+- **Data scanned by the query**. This is the primary driver of cost and is incurred every time the monitor is evaluated. To reduce costs, optimize your query using [default scope](/docs/manage/partitions/flex/faq/#how-can-i-optimize-my-query-using-default-scope) to include only necessary partitions and minimize the amount of data scanned.
+- **Time range of the monitor query**. For static monitors, adjust the detection window under [Trigger Type](/docs/alerts/monitors/create-monitor/#step-1-set-trigger-conditions) (for example, `"Alert when result is _____ within <detection window> minutes"`) to use a shorter time range, which reduces the amount of data scanned. For outlier monitors, reduce the **datapoints** parameter under **Trigger Type** to lower the scanned bytes.
+- **Frequency of monitor evaluation**. Currently, the frequency at which a monitor is evaluated is not configurable. However, it's essential to be mindful that more frequent evaluations will lead to higher scan costs.
+
+By carefully configuring these elements, you can optimize monitor performance while keeping scan costs under control.
```

Comment:
By carefully configuring these elements, you can balance scan costs with monitoring requirements.

Reply:
perfect! fixed.